glean-core/android/src/test/java/mozilla/telemetry/glean/private/LabeledMetricTypeTest.kt

```diff
@@ -23,40 +17,17 @@ import org.junit.Assert.assertFalse
 import org.junit.Rule
 /*
 import java.util.UUID
```

Comment:
Let's remove this too. We're past the point in which we could simply do a file diff with a-c. If needed, we can add it back.